src/sequence.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  FindRoute,
+  inject,
+  InvokeMethod,
+  parseOperationArgs,
+  ParsedRequest,
+  Reject,
+  Send,
+  ServerResponse,
+  SequenceHandler,
+} from '@loopback/core';
+
+import {
+  AuthenticateFn,
+} from '@loopback/authentication';
+
+export class MySequence implements SequenceHandler {
+  constructor(
+    @inject('sequence.actions.findRoute') protected findRoute: FindRoute,
+    @inject('sequence.actions.invokeMethod') protected invoke: InvokeMethod,
+    @inject('sequence.actions.send') protected send: Send,
+    @inject('sequence.actions.reject') protected reject: Reject,
+    @inject('authentication.actions.authenticate')
+    protected authenticateRequest: AuthenticateFn,
+  ) {}
+
+  async handle(req: ParsedRequest, res: ServerResponse) {
+    try {
+      const route = this.findRoute(req);
+
+      fakeAuthenticate(req);
```

Comment:
I find this confusing. IIUC, the goal here is to inject authentication header into the request. In that case, please rename the method to something like `setFakeAuthorizationHeader`.